### PR TITLE
Translation updates for Marathi, Swahili, Igbo, Japanese, Somali, Afrique, Arabic, and Korean

### PR DESCRIPTION
**Pull request by jimjohnsonrollings**
Resolves N/A

**Overall change:**
A bunch of translation updates

**Code changes:**

- Full set of updates for Marathi
- Full set of updates for Swahili, Somali, Afrique, Arabic, and Korean ahead of Front Page launch
- Some updates for Japanese
- Commas added in for Igbo
- Integration tests updated to pass for Korean, and to rename a variable to more closely describe what it's testing so that I don't get confused in the future

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] I have added labels to this PR for the relevant pod(s) affected by these changes
- [x] I have assigned this PR to the Simorgh project

**Testing:**

- [x] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [ ] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`)
- [ ] This PR requires manual testing

### CODE REVIEW

File: src/app/lib/config/services/marathi.js [hunk 1]
```diff
@@ -62,17 +62,17 @@ export const service = {
...
           solutions: [
             'तुमचं URL पुन्हा एकदा तपासा',
             'तुमच्या ब्राऊझरमध्ये रिफ्रेश बटण दाबा',
             'बीबीसीचं सर्च बार वापरून हे पृष्ठ शोधा',
           ],
           callToActionFirst: 'किंवा, ',
-          callToActionLinkText: 'बीबीसी न्यूज मराठी]च्या',
-          callToActionLast: ' होमपेजला भेट द्या.',
+          callToActionLinkText: 'बीबीसी न्यूज मराठी',
+          callToActionLast: 'च्या होमपेजला भेट द्या.',
```
Comment (jimjohnsonrollings) [inline src/app/lib/config/services/marathi.js:75]: Although different from other services, it seems that it is correct in Marathi that some suffixes should fall outside the hyperlink, which is why there is no space at the beginning of this string. There are a couple of other instances in this file too.

File: src/app/lib/config/services/marathi.js [hunk 2]
```diff
@@ -153,27 +152,27 @@ export const service = {
         },
         listen: 'ऐका',
         watch: 'पाहा',
-        liveLabel: 'थेट',
-        nextLabel: 'NEXT',
+        liveLabel: 'LIVE',
```
Comment (jimjohnsonrollings) [inline src/app/lib/config/services/marathi.js:155]: I've checked and they definitely want this in English

**Review by DenisHdz — APPROVED** [review]

**Review by OlgaLyubin — APPROVED** [review]
LGTM

File: src/app/lib/config/services/somali.js [hunk 3]
```diff
@@ -160,26 +158,26 @@ export const service = {
...
           text: 'Macluumaadkan lama heli karo',
           linkText: 'Faahfaahin ka eeg %provider_name%',
-          linkTextSuffixVisuallyHidden: ', bogag kale',
+          linkTextSuffixVisuallyHidden: ', Bogag kale',
           warningText:
-            'BBC masuul kama ahan macluumadka bogagga kale ee dibadda.',
+            'BBC masuul kama ahan macluumadka bogagga kale ee dibadda.',
         },
         skipLink: {
           text: 'Skip %provider_name% post',
```
Comment (jimjohnsonrollings) [inline src/app/lib/config/services/somali.js]: Awaiting updated translations for Somali skip link and end of skip

File: src/app/lib/config/services/burmese.js [hunk 4]
```diff
@@ -160,7 +160,7 @@ export const service = {
         },
         listen: 'နားဆငျရနျ',
         watch: 'ကြည့်ရှုရန်',
-        liveLabel: 'တိုက်ရိုက်ထုတ်လွှင့်မှု',
+        liveLabel: 'တိုက်ရိုက်',
```
Comment (jimjohnsonrollings) [inline src/app/lib/config/services/burmese.js]: Awaiting confirmation from editorial

**Review by OlgaLyubin — COMMENTED** [review]
Jenkins is complaining about snapshots failing

**Comment by jimjohnsonrollings** [comment]
I think I've made all the tests pass!

**Review by sareh — APPROVED** [review]
👍 Looks good. Thanks for the updates.

**Review by j-pendlebury — APPROVED** [review]

**Review by tochwill — APPROVED** [review]

**Review by DenisHdz — APPROVED** [review]

**Pull request merged**